### PR TITLE
Artifact list column creation time

### PR DESCRIPTION
**Pull request by Bobgy**
Implements feature mentioned in https://github.com/kubeflow/pipelines/issues/2086#issue-492003078

> We should add a column showing the creation time of each artifact. In order to get this, we'll need to query the metadata API server for events, and find the associated event that produced any given artifact. The event will have a timestamp.

Implementation video: https://drive.google.com/file/d/1tk6rFIuFn2H3yt9rwA7RYOPj0Loxhq_P/view

Current trade off, we need to send an extra request for each artifact to get its creation time. Therefore
* this is not scalable when artifact count is large.
* this shows data very slowly only until all the data is ready.

Thoughts?
Ideal fix is letting backend provide an api that lists artifacts with their creation time.
Before that, I can probably first show other fields, and then show creation time when data is ready.

/area front-end

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2105)

### CODE REVIEW

File: frontend/start-dev.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+function clean_up() {
+  kill -15 %1
+  kill -15 %2
+  kill -15 %3
+}
+
+echo "Preparing dev env for KFP frontend"
+
+echo "Detecting api server pod names..."
+MLMD_ENVOY_POD=$(kubectl get pods -l component=metadata-envoy -o=custom-columns=:.metadata.name --no-headers)
```
Comment (Ark-kun) [inline frontend/start-dev.sh]: This line does not seem to be passing the namespace. Can this be a problem?

Comment (Bobgy) [reply frontend/start-dev.sh]: Good catch! Thanks. I should add the namespace.

I've been using kubeflow namespace as default, so I overlooked this one.
BTW, these changes are from a separate PR: https://github.com/kubeflow/pipelines/pull/2104

**Comment by Bobgy** [comment]
@neuromage Do you have any feedback on the demo video?

**Comment by Bobgy** [comment]
Note, this PR is based on #2104, I will merge that one first

**Comment by neuromage** [comment]
/lgtm
/approve

Thanks @Bobgy

**Pull request merged**